google/cloud/bigtable/internal/async_row_sampler_test.cc

```diff
@@ -438,6 +450,54 @@ TEST(AsyncSampleRowKeysTest, CurrentOptionsContinuedOnRetries) {
   timer_promise.set_value(make_status_or(std::chrono::system_clock::now()));
 }
 
+TEST_F(AsyncSampleRowKeysTest, BigtableCookie) {
```

Comment:
Maybe add a comment saying the validation happens in the metadata fixture?  It kind of makes it hard to see what the test is doing

Reply:
Added clarifying comments which should draw the reader's eye.